Website/Packages/datax-pipeline/src/modules/flowDefinition/flowActions.js

```diff
@@ -102,6 +110,12 @@ export const initFlow = context => (dispatch, getState) => {
     } else {
         const owner = UserSelectors.getUserAlias(getState());
         dispatch(QueryActions.initQuery(QueryModels.defaultQuery));
+        isDatabricksSparkType().then(response => {
```

Comment:
I think it'd be better to call this after FLOW_NEW dispatch call.

Reply:
Cannot call it after flow new since new flow already gets created. Also query is iniitalized before new flow